pages/resume.tsx

```diff
@@ -1,22 +1,83 @@
 import React, { FC } from 'react';
 import Resume from '../components/resume/resume';
-import jbatorski from '../data/resume/jbatorski.json';
+import { Basics, Job } from '../components/resume/types';
+import resumeJson from '../data/resume/resume.json';
+import { ResumeJson, WorkItem } from '../data/resume/types';
 
 interface ResumePageProps {
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    data: any
+    basics: Basics;
+    skills: string[];
+    jobs: Job[];
 }
 
-const ResumePage: FC<ResumePageProps> = ({ data }) => {
-    return <Resume data={data} />;
+const ResumePage: FC<ResumePageProps> = uiModel => {
+    return <Resume {...uiModel} />;
 };
 
 export async function getStaticProps() {
+    const uiModel = transform(resumeJson as ResumeJson);
+
     return {
-        props: {
-            data: jbatorski,
+        props: uiModel,
+    };
+}
+
+export default ResumePage;
+
+function transform({ basics, skills, work }: ResumeJson) {
```

Comment:
This should really be in it's own file, I just don't know where to put it since it spans the data and UI models. So punting on the decision for now and will revisit if I ever get around to allowing an "upload your own json resume" feature.